ports/mimxrt10xx/Makefile

```diff
@@ -97,7 +103,14 @@ else
 endif
 
 flash-sdp: $(BUILD)/$(OUTNAME).bin
-	@if [ -z "$(SDPHOST)" ]; then echo SDPHOST is not found for this machine; exit 1; fi
+	@:$(call check_defined, SDPHOST, not found for this machine)
+	$(SDPHOST) -u 0x1fc9,$(SDP_$(MCU)_PID) -- write-file $(SDP_$(MCU)_WRITE_ADDR) $<
+	$(SDPHOST) -u 0x1fc9,$(SDP_$(MCU)_PID) -- jump-address $(SDP_$(MCU)_JUMP_ADDR)
+
+# TODO write-register doesn't work
+erase-app: $(BUILD)/$(OUTNAME).bin
+	@:$(call check_defined, SDPHOST, not found for this machine)
+	$(SDPHOST) -u 0x1fc9,$(SDP_$(MCU)_PID) -- write-register $(DBL_TAP_REG_ADDR) 32 $(DBL_TAP_MAGIC_ERASE_APP)

```

Comment:
@nxf58843 somehow I couldn't get the write-register to work, it always get no response from device. Though this is not important, most user should use the erase-firmware.uf2 . I just wonder if I made any mistakes. Else we could just remove this together. 

```
sdphost -u 0x1fc9,0x0145 -- write-register 0x400D410C 32 0xf5e80ab4
getStatusResponse.readPacket error 5.
Status (HAB mode) = 10004 (0x2714) No response from device.
```

read-register doesn't work as well

```
sdphost -u 0x1fc9,0x0145 -- read-register 0x400D410C
getStatusResponse.readPacket error 5.
Status (HAB mode) = 10004 (0x2714) No response from device.
```

Reply:
Let me look into this.
Have you tried reading an address from OCRAM?
I need to see if that address is accessible or needs to be enabled.

Reply:
I have just tried to read from dtcm and ocram but it doesn't response either. No problems at all, I am just curious, using uf2 is more preferable. I will merge it now, you don't have to "fix" it at all :smiley: 

```
$ sdphost -u 0x1fc9,0x0145 -- read-register 0x20200000
getStatusResponse.readPacket error 5.
Status (HAB mode) = 10004 (0x2714) No response from device.

$ sdphost -u 0x1fc9,0x0145 -- read-register 0x20000000
getStatusResponse.readPacket error 5.
Status (HAB mode) = 10004 (0x2714) No response from device.
```

Reply:
Glad you can move forward with the uf2 implementation.
It looks like the read/write register commands were dropped from the RT1010 due to ROM size limitation.
You could write a single word using write-file, but adding another file is not much better than separate binaries.
I may try setting the magic numbers with different jump addresses when I can find time.

Reply:
ah thanks for confirmation, I will just remove the make target. uf2 will be the standard way for erase_firmware (factory reset) for tinyuf2. Don't worry about having other register for erase_firmware. I don't see any issue with packing to the double tap reg. May only need to rename these variable a bit to be more generic than just double tap